cli/functionary/login.py

```diff
@@ -16,15 +17,7 @@ def login_cmd(ctx, user, password, host):
     for other functionary commands to use to communicate with the server.
     """
     login_url = f"{host}/api/v1/api-token-auth"
-    success, message = login(login_url, user, password)
-
-    # check status code/message on return then exit
-    if success:
-        click.echo("Login successful!")
-    else:
-        click.secho(
-            message,
-            err=True,
-            fg="red",
-        )
-        ctx.exit(1)
+    token = login(login_url, user, password)
+    click.echo("Login successful!")
```

Comment:
This message should come after the `save_config_value` calls, since if one of those calls fails, the fact that the login succeeded doesn't really help.